test/integration/sign-out-test.js

```diff
@@ -34,7 +29,7 @@ test.skip('sign out', function (t) {
   })
 
   account.on('signout', function () {
-    t.is(account.isSignedIn(), false)
```

Comment:
the test was correct, `account.isSignedIn()` should return `false`, not `true`, when the `signout` event gets triggered. You have to fix the implementation, not the test. The `'signout'` event gets emitted here: https://github.com/hoodiehq/hoodie-account-client/blob/master/lib/sign-out.js#L41, the `account.isSignedIn()` method is implemented here: https://github.com/hoodiehq/hoodie-account-client/blob/master/lib/is-signed-in.js. It’s something with these two that needs to be changed so that this test passes